src/components/views/elements/ImageView.js

```diff
@@ -216,7 +216,7 @@ export default class ImageView extends React.Component {
                                 { this.getName() }
                             </div>
                             { eventMeta }
-                            <a className="mx_ImageView_link" href={ this.props.src } download={ this.props.name } target="_blank" rel="noopener">
+                            <a className="mx_ImageView_link" href={ this.props.src } download={ this.props.name } rel="noreferrer noopener">
```

Comment:
why are we removing the target=_blank here?

Reply:
> I'm not sure what part of this PR does the thing in the title?

that is that part. `download` overrides behaviour such that it being sent into a new tab is not needed and actually breaks downloading in electron because we send new tabs into your browser so we don't handle downloading locally.

Reply:
aaaah, I see